config/src-3/bloop/config/SourceGeneratorCompanionPlatform.scala

```diff
@@ -0,0 +1,3 @@
+package bloop.config
+
+trait SourceGeneratorCompanionPlatform {}
```

Comment:
thought: `extends AbstractFunction3` was causing a "missing apply method" in Scala 3. Adding that `apply` was conflicting with the other `apply` - hence, version-specific sources.